compiler/rustc_mir_build/src/thir/cx/expr.rs

```diff
@@ -570,6 +570,9 @@ impl<'tcx> Cx<'tcx> {
 
             hir::ExprKind::ConstBlock(ref anon_const) => {
                 let anon_const_def_id = self.tcx.hir().local_def_id(anon_const.hir_id);
+
+                // FIXME Do we want to use `from_inline_const` once valtrees
```

Comment:
I think that function is only used from here? . If so, we can move it onto constantkind and turn inline consts always into mir constants

Reply:
Yes this is only used from here. But changing `from_inline_const` would require some changes in how we process patterns during thir construction. The next PR will be about changing thir to use `mir::Constant` instead of `ty::Const`, so would also include the change to `from_inline_const`. I would prefer to leave this as is in this PR, unless you want this change right now.